appveyor.yml

```diff
@@ -11,6 +11,10 @@ environment:
        SONARCLOUD_URL: 'https://sonarcloud.io'
        SONARCLOUD_TOKEN:
          secure: th/5nKHJ0jW1sM3a0bs0cohRLIeMAx4Y5Do/al3L11XbWHOgoCdNYrNBQmSo9tFs
+       # To be changed in the future since this token belongs to a personal account. See below.
+       # https://github.com/RHEAGROUP/DEHP-Common/wiki/Build-And-Release#modifying-the-github-token
+       GITHUB_TOKEN:
+         secure: uS7/WlBplrAs/j9ab9P3SUN7GZyf4QAHZX//YpDOzXG/DZ6o1MGHdIWLHHdDYM5P
```

Comment:
this should indeed **not** be here... please add a ticket somewhere so this is captured that work needs to be done ASAP